### PR TITLE
[stable/prometheus] fix remoteRead/Write defaults

### PR DESCRIPTION
**Pull request by flecno**
#### What this PR does / why we need it:
#20056 has added the options to set `remote_read` and `remote_write` configs but ueses a wrong standard value.
Using this value as list works but shows this warning:

```
Warning: Merging destination map for chart 'prometheus'. Cannot overwrite table item 'remoteRead', with non table value: map[]
```

The config is a list of objects not an object:

https://prometheus.io/docs/prometheus/latest/configuration/configuration/#configuration-file
```
# Settings related to the remote write feature.
remote_write:
  [ - <remote_write> ... ]

# Settings related to the remote read feature.
remote_read:
  [ - <remote_read> ... ]
```

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped]
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name

### CODE REVIEW

**Review by zanhsieh — APPROVED** [review]

File: stable/prometheus/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: prometheus
-version: 11.1.2
+version: 11.1.3
```
Comment (zanhsieh) [inline stable/prometheus/Chart.yaml]: Should be 11.1.4 for now @flecno

Comment (flecno) [reply stable/prometheus/Chart.yaml]: Ok, I have bumped the version and rebased to master

**Comment by flecno** [comment]
@zanhsieh can you review again and merge/release it? 😃

**Review by zanhsieh — APPROVED** [review]

File: stable/prometheus/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: prometheus
-version: 11.1.3
+version: 11.1.4
```
Comment (zanhsieh) [inline stable/prometheus/Chart.yaml]: I think the e2e test failed due to version problem. Should be 11.1.5.

**Review by zanhsieh — CHANGES_REQUESTED** [review]
Please bump Chart version to latest.

**Comment by flecno** [comment]
Bumped again :)

**Pull request merged**